ChangeLog

```diff
@@ -1,3 +1,177 @@
+3.12.0b1:
+	New Features:
+	- Add an --key-type option to specify RSA key size to cf-key
+	- New hash_to_int policy function (CFE-2733)
+	- Issue a warning on ignored locking attributes (CFE-2748)
+	- Add IPv6 hard classes with the "ipv6_" prefix (CFE-2310)
+
+	Changes:
+	- Load augments at the end of context discovery
+	  This means that classes defined as part of the context discovery
+	  (e.g. 'am_policy_hub' and 'policy_server') can be used in the
+	  augments. (CFE-2482)
+	- Disallow modifications of variables from a remote bundle (CFE-1915)
+	- Speedup evalution by not copying variables table when expanding a promise
+	  (CFE-2524)
+	- Resolve subkey conflicts when converting to JSON
+	  Whenever there is a conflict of array variable definitions prefer
+	  the container subkeys over simple values when converting to JSON.
+	  (CFE-2536)
+	- Do not ignore meta promises in server bundles (CFE-2066)
+	- Add a debug log for computed class in splayclass
+	- Dynamically grow HashMap on inserts (CFE-2301)
+	- Dynamically shrink HashMap when items are removed (CFE-2301)
+	- Don't error when calling isexecutable on broken link (CFE-741)
+	- Open measurements promise type from enterprise cf-monitord.
```

Comment:
Looks weird that some lines end in dot, while others don't. Remove period here, as well as in other places.